src/pages/home.js

```diff
@@ -1,32 +1,69 @@
 import styled from 'styled-components';
-import BackgroundImage from '../images/Background.jpg';
 import Statistics from '../components/Statistics/Statistics';
+import CreateDiver from '../components/FormCreateDiver/CreateDiver';
+import { useState } from 'react';
+import { useLocalStorage } from 'usehooks-ts';
+import DiverInfo from '../components/DiverInfomation/DiverInfo';
 
 export default function Home({ diveData }) {
+  const [createDiverPageActive, setCreateDiverPageActive] = useState(false);
+  const [diverInfo, setDiverInfo] = useLocalStorage('diverInfo', []);
+
   return (
     <Wrapper>
-      <Headline>logbuddy</Headline>
-      <Statistics diveData={diveData} />
+      {diverInfo[0]
+        ? diverInfo?.map(
+            ({ name, organization, date, certification, cert_nr, _id }) => (
+              <DiverInfo
+                key={_id}
+                name={name}
+                organization={organization}
+                date={date}
+                certification={certification}
+                certnr={cert_nr}
+              />
+            )
+          )
+        : !createDiverPageActive && (
+            <Header onClick={handleCreateDiverPage}>create cew diver +</Header>
+          )}
```

Comment:
I'm not sure about this part: Is there always only one object in your state `diverInfo`? If so then you don't need to save this object in an array as you only want to render one `DiverInfo` component? If this is true then you don't need to map the data, just pass them as props and use them in your component. :)